src/lib.rs

```diff
@@ -85,9 +85,14 @@ pub use util::address::AddressType;
 pub use util::amount::Amount;
 pub use util::amount::Denomination;
 pub use util::amount::SignedAmount;
+pub use util::merkleblock::MerkleBlock;
+
+pub use util::ecdsa;
+pub use util::schnorr;
```

Comment:
:+1: 

This is a tiny step towards https://github.com/rust-bitcoin/rust-bitcoin/pull/525 where the `util` module was made private.